Applications/WebPortal/WebApp/library/source/pages/account_directory_page/local_account_directory_model.ts

```diff
@@ -17,7 +17,9 @@ export class LocalAccountDirectoryModel extends AccountDirectoryModel {
     this._groups = groups.clone();
     this._accounts = new Beam.Map<Beam.DirectoryEntry, AccountEntry[]>();
     for(const group of this._groups) {
-      this._accounts.set(group, accounts.get(group).slice());
+      if(accounts.get(group)) {
```

Comment:
I don't think this check should be performed.

Reply:
It is possible that some groups might not have any accounts associated with them....

Reply:
Yes that's fine, but no error would get thrown in that case.

Reply:
So every group should be in the map even if the array associated with it is empty?

Reply:
Yep, that means the group is empty. The UI indicates how to handle an empty group.